service/scaladsl/kafka/client/src/main/scala/com/lightbend/lagom/internal/scaladsl/broker/kafka/ScaladslKafkaSubscriber.scala

```diff
@@ -164,16 +164,20 @@ private[lagom] class ScaladslKafkaSubscriber[Payload, SubscriberPayload](
       )
 
     val backoffConsumerProps =
-      BackoffSupervisor.propsWithSupervisorStrategy(
-        consumerProps,
-        s"KafkaConsumerActor$consumerId-${topicCall.topicId.name}",
-        consumerConfig.minBackoff,
-        consumerConfig.maxBackoff,
-        consumerConfig.randomBackoffFactor,
-        SupervisorStrategy.stoppingStrategy
-      )
-
-    system.actorOf(backoffConsumerProps, s"KafkaBackoffConsumer$consumerId-${topicCall.topicId.name}")
+      BackoffOpts
+        .onStop(
+          consumerProps,
+          s"KafkaConsumerActor$consumerId-${topicCall.topicId.name}",
+          consumerConfig.minBackoff,
+          consumerConfig.maxBackoff,
+          consumerConfig.randomBackoffFactor
+        )
+        .withDefaultStoppingStrategy
```

Comment:
I'm not familiar with the idea of stopping message in `BackoffOpts` but IIUC, since `ScaladslKafkaSubscriber`,  is not sharded we don't need an extra stopping message.

PS: I've noticed the refactor to `BackoffOpts` in 4 places in this PR and the use (or not) of `withDefaultStoppingStrategy` was a bit confusing.

Reply:
> PS: I've noticed the refactor to `BackoffOpts` in 4 places in this PR and the use (or not) of `withDefaultStoppingStrategy` was a bit confusing.

Yeah, my mistake. It is consistent now.